raiden/transfer/mediated_transfer/initiator.py

```diff
@@ -74,23 +74,27 @@ def handle_block(
     secrethash = initiator_state.transfer.lock.secrethash
     locked_lock = channel_state.our_state.secrethashes_to_lockedlocks.get(secrethash)
 
-    lock_expiration_threshold = locked_lock.expiration + DEFAULT_NUMBER_OF_BLOCK_CONFIRMATIONS * 2
-    lock_has_expired, _ = locked_lock and channel.is_lock_expired(
-        end_state=channel_state.our_state,
-        lock=locked_lock,
-        block_number=state_change.block_number,
-        lock_expiration_threshold=lock_expiration_threshold,
-    )
-    if lock_has_expired:
-        expired_lock_events = channel.events_for_expired_lock(
-            channel_state,
-            secrethash,
-            locked_lock,
-            pseudo_random_generator,
+    if locked_lock:
```

Comment:
Is this the bug you meant this is fixing?  So if the secrethash is not in the secrethashes to locked locks mapping?

Note: I would do:

```
if not locked_lock:
    return TransitionResult(initiator_state, list())

...
...
```

This way you don't need to have an extra level of indentation.

Reply:
Yes, and this code also had problems `locked_lock and channel.is_lock_expired(`